corehq/apps/api/urls.py

```diff
@@ -72,24 +74,23 @@
         ReportResource,
         v0_5.DeviceReportResource,
         DomainMetadataResource,
+        locations.v0_5.LocationResource,
+        locations.v0_5.LocationTypeResource,
```

Comment:
Not 100% sure on how this versioning is supposed to work.  Our code refers to the v3 location resource in a few places, so that's where I stuck the old version.  I figure the new one goes in v5, but not sure if there's any point in sticking either in the other API versions.